be/src/runtime/exec_env.cpp

```diff
@@ -134,13 +134,7 @@ Status ExecEnv::_init(const std::vector<StorePath>& store_paths) {
     _thread_pool = new PriorityThreadPool("olap_scan_io", // olap scan io
                                           config::doris_scanner_thread_pool_thread_num,
                                           config::doris_scanner_thread_pool_queue_size);
-    _pipeline_scan_io_thread_pool = new PriorityThreadPool("pip_scan_io", // pipeline scan io
```

Comment:
Why we could remove this thread pool ?

Reply:
We use `io_dispatcher_thread_pool` for now.